### PR TITLE
gSender job loss recovery update

### PR DESCRIPTION
**Pull request by StephenGitHuber**

### CODE REVIEW

**Comment by SienciLabs** [comment]
Ok I'll begin looking at it

**Comment by SienciLabs** [comment]
@StephenGitHuber, take a look at the [rearranged work I did on your work](https://github.com/Sienci-Labs/Resources/pull/18/commits/d60f527600dc344c9387842fc42c936b8a9811e8). I thought your writing was a great elaboration on the feature, but what I found confusing was it seemed to imply that the feature was only useful for resuming long carves and also I felt there were some other details that were repeated or could be made clearer.

I'd love to have you review it and let me know your thoughts on if we want to tweak it further

File: _images/_gsender/_using/gs_us_job-loss1.jpg
Comment (StephenGitHuber) [inline _images/_gsender/_using/gs_us_job-loss1.jpg:1]: Looks good

File: gsender/gs-using-gsender.md [hunk 1]
```diff
@@ -164,50 +164,43 @@ Now you're off and cutting, what a thrill! While your job is running keep an eye
 Also called “Start from Line”, this feature can recover a carve you were working on that:
 
 1. Was disrupted by a power loss, USB disconnect, mechanical malfunction, or other failure so you need to resume from where it failed
-1. Was really long, so you want to resume carving after you paused it for the night
+1. Was really long, so you want to resume carving after [pausing it for the night](#steps-to-pause-cutting-mid-job)
 1. You only wanted to run part of the job anyway
 
+This will work whether it's your first time running the file, you've reloaded it, you stopped it manually, it got disconnected somehow, or any other situation.
 
+We worked hard to make it account for typical ‘setup’ g-code that is at the start of your file like units, zeros, spindle speed or laser power, track all machine movements in the file to locate exactly where it needs to resume from, and also run gSenders automatic g-code commands. This way you can be confident in returning to your projects even when something goes awry.
```
Comment (StephenGitHuber) [inline gsender/gs-using-gsender.md:172]: This is quite a lot for the user to think about, and may cause some anxiety, as it is a complex thing that is happening here. I initially glazed over it until I got to 'track all movements in the file'. I went back and slowed down to read 'units'....hmm ok like mm or inches. Then I go to 'zeros', so it knows where I set the zero, but am I there now? Spindle speed or laser power, well I don't even have those. Back to the 'track all machine movements part' and I'm unsure of where I ended, where my zero is/should be and where it's going. I may be a bit too analytical here, but let me try and offer a couple changes for you to consider.

**Review by SienciLabs — APPROVED** [review]
Think the overall addition does a much better job explaining the feature and all the scenarios it should be used!

**Pull request merged**